SecureAuth.Sdk.UnitTests/App.config

```diff
@@ -7,6 +7,8 @@
         <add key="ApiKey" value=""/>
 
       <!-- Authentication Service Test Values -->
+        <add key="UserSvc.goodDomain" value="corporate"/>
+        <add key="UserSvc.badDomain" value="guest"/>

```

Comment:
duplicate appsetting with line 22?

Reply:
Ah, yeah.  Copy and paste error.  Should be AuthSvc.